harness/determined/experimental/core_v2/_core_v2.py

```diff
@@ -140,42 +93,34 @@ def _init_context(
         )
         return _context
 
-    # Unmanaged trials.
-    if defaults is None:
-        raise NotImplementedError(
-            "either specify `defaults`, or run as a managed determined experiment"
-        )
-
     # Construct the config.
-    defaulted_config = defaults or DefaultConfig()
-    unmanaged_config = unmanaged or UnmanagedConfig()
-    checkpoint_storage = checkpoint_storage or defaulted_config.checkpoint_storage
-
-    config = {
-        "name": defaulted_config.name or f"unmanaged-{uuid.uuid4().hex[:8]}",
-        "data": defaulted_config.data,
-        "description": defaulted_config.description,
-        "labels": defaulted_config.labels,
-        "searcher": defaulted_config.searcher
-        or {
-            "name": "single",
-            "metric": "unmanaged",
-            "max_length": 100000000,
-        },
-        "workspace": unmanaged_config.workspace,
-        "project": unmanaged_config.project,
-    }
-
-    config_text = util.yaml_safe_dump(config)
+    config = unmanaged_config or UnmanagedConfig()
```

Comment:
also, we should have warnings for what we override in each case (cluster/managed)

Reply:
Sure, warning added